task.aspx

```diff
@@ -88,13 +95,13 @@
 
                   <div class="row">
                      <div class="col-4">
-                        <asp:Button ID="Button1" class="btn btn-lg btn-block btn-success" runat="server" Text="Add" />
+                        <asp:Button ID="Add" class="btn btn-lg btn-block btn-success" runat="server" Text="Add" OnClick="BttnAdd_Click" />
                      </div>
                      <div class="col-4">
-                        <asp:Button ID="Button3" class="btn btn-lg btn-block btn-warning" runat="server" Text="Update" />
+                        <asp:Button ID="Update" class="btn btn-lg btn-block btn-warning" runat="server" Text="Update" OnClick="BttnUpdate_Click" />
```

Comment:
### Auto-generated events

Again, Visual Studio will generate events automatically for you based on the ID you've given and the event you selected.

By right this would look like 
```suggestion
                        <asp:Button ID="btnUpdate" class="btn btn-lg btn-block btn-warning" runat="server" Text="Update" OnClick="btnUpdate_Click" />
```